### ISSUE TITLE
Add accessibility label to Other Donation Amount Input field

### ISSUE THREAD

**Issue by bryancasler** [4site-interactive-studios/engrid-scripts#144]
The Other Donation Amount Input field does not have a label, and it shouldn't have a visible label. But! it could get a screen-reader accessible label using "aria-label".

This:
```
<div class="en__field--donationAmt">
...
<input name="transaction.donationAmt.other" placeholder="Other">
```

Becomes:
```
<div class="en__field--donationAmt">
...
<input name="transaction.donationAmt.other" placeholder="Other" aria-label="Enter your custom donation amount">
```

We should provide defaults in ENgrid and clients can override this on a per theme basis.

Accessibility checker: https://webaim.org/

**Comment by bryancasler** [comment]
Added https://github.com/4site-interactive-studios/engrid-scripts/commit/56f6dceb9619f8276010f952a6dd53ad1bd6fa85

No way to override it, that can be done in the future if needed
